lesson2/random-massive.py

```diff
@@ -0,0 +1,14 @@
+import random
+lst1 = list()
+
+
+def randomlist(minr, maxr, lstx=lst1, lenlst=20):
```

Comment:
Function should start with verb.
```suggestion
def generate_random_list(min_r, max_r, list_length=20):
```